packages/astro/src/core/build/plugins/index.ts

```diff
@@ -18,7 +17,6 @@ import { pluginSSR, pluginSSRSplit } from './plugin-ssr.js';
 
 export function registerAllPlugins({ internals, options, register }: AstroBuildPluginContainer) {
 	register(pluginComponentEntry(internals));
-	register(pluginAliasResolve(internals));
```

Comment:
This is a silent change that's not documented in the changesets as it's fixed from a Vite 5 upgrade. Users shouldn't notice importing components with aliases not working.